doc/src/build/fullnode.md

```diff
@@ -1,58 +1,47 @@
+title: Run a Sui Full node
 ---
 
-Sui full nodes run a service that
-stores the full blockchain state and history. They service reads, either for
-end clients or by helping other full nodes get up-to-date with the latest
-transactions that have been committed to the chain.
+Sui Full nodes run a service that stores the full blockchain state and history. They service reads, either for end clients or by helping other Full nodes get up-to-date with the latest transactions that have been committed to the chain.
 
-This role enables
-[validators](../learn/architecture/validators.md) (or miners in
-other networks) to focus on servicing the write path and processing
-transactions as fast as possible. Once a validator has committed a new set of
-transactions (or a block of transactions), the validator will push that block
-to a full node (potentially a number of full nodes) who will then in turn
-disseminate it to the rest of the network.
+This role enables [validators](../learn/architecture/validators.md) (or miners in
+other networks) to focus on servicing the write path and processing transactions as fast as possible. Once a validator has committed a new set of transactions (or a block of transactions), the validator will push that block to a Full node (potentially a number of Full nodes) who will then in turn disseminate it to the rest of the network.
 
 ## Features
 
-Sui full nodes exist to:
+Sui Full nodes:
 
 * Track and verify the state of the blockchain, independently and locally.
 * Serve read requests from clients.
 * Conduct local app testing against verified data.
 
 ## State synchronization
 
-Today Sui full nodes sync with validators to be able to learn about newly
-committed transactions.
+Sui Full nodes sync with validators to receive newly committed transactions.
 
-The normal life of a transaction requires a few round trips to 2f+1 validators
-to be able to form a TxCert, at which point a transaction is guaranteed to be
-committed and executed.
+A transaction requires a few round trips to 2f+1 validators to be able to form a TxCert, at which point a transaction is guaranteed to be committed and executed.
 
-Today, this synchronization process is performed by:
+This synchronization process is performed by:
 
 1. Following 2f+1 validators and listening for newly committed transactions.
 1. Requesting the transaction from one validator.
 1. Locally executing the transaction and updating the local DB.
 
 This synchronization process is far from ideal as it requires listening
-to at a minimum 2f+1 validators to ensure that a full node has properly seen all
+to at a minimum 2f+1 validators to ensure that a Full node has properly seen all
 new transactions. Over time, we will improve this process (e.g. with the
-introduction of checkpoints, ability to synchronize with other full nodes,
-etc.) in order to have better guarantees around a full node’s ability to be
+introduction of checkpoints, ability to synchronize with other Full nodes,
```

Comment:
checkpoints is already there. cc @lxfind / @mystenmark  to provide a paragraph here